server/src/pages/shotindex/view.js

```diff
@@ -91,6 +95,13 @@ class Body extends React.Component {
     );
   }
 
+  getShotType(dimensions) {
+    const squareBuffer = 50;
```

Comment:
There should be some comment about what this values means and why it is 50.  Maybe "An image is 'square' if it's within squareBuffer pixels of being square"?